clang/test/Driver/sycl-bfloat16-lib.cpp

```diff
@@ -123,7 +123,9 @@
 // BFLOAT16-FALLBACK-NEXT: clang-offload-bundler{{.*}} "-type=o" "-targets=sycl-{{spir64_gen-|spir64-}}unknown-unknown" "-input={{.*}}libsycl-fallback-imf-bf16.o" "-output={{.*}}libsycl-fallback-imf-bf16-{{.*}}.o" "-unbundle"
 // BFLOAT16-FALLBACK-NEXT: clang-offload-bundler{{.*}} "-type=o" "-targets=sycl-{{spir64_gen-|spir64-}}unknown-unknown" "-input={{.*}}libsycl-fallback-bfloat16.o"
 
-// BFLOAT16-NONE-NATIVE:      clang-offload-bundler{{.*}} "-type=o" "-targets=sycl-spir64-unknown-unknown" "-input={{.*}}libsycl-crt.o" "-output={{.*}}libsycl-crt-{{.*}}.o" "-unbundle"
+// BFLOAT16-NONE-NATIVE: clang{{.*}} "-triple" "spir64-unknown-unknown"
+// BFLOAT16-NONE-NATIVE-NEXT: llvm-link{{.*}}
+// BFLOAT16-NONE-NATIVE-NEXT: clang-offload-bundler{{.*}} "-type=o" "-targets=sycl-spir64-unknown-unknown" "-input={{.*}}libsycl-crt.o" "-output={{.*}}libsycl-crt-{{.*}}.o" "-unbundle"
```

Comment:
I think we should drop all these checks. The comment at the top says:

```
///
/// Check if bfloat16 native and fallback libraries are added on Linux
///
```
I suggest we live only relevant checks (i.e. matching commands handling bfloat16 libraries only) and remove all other checks. It should improve readability and simplify maintenance. I suppose this change doesn't impact bfloat16 libraries, so ideally, we should pass this test with no test changes.

Reply:
I have updated the test to only check for bfloat16 inclusions